src/components/YourNameComponent.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import style from "./Home.module.css";
+
+const YourNameComponent = ({ name, onNameChange }) => {
+  useEffect(() => {
+    const storedName = localStorage.getItem('name');
```

Comment:
should probably have something more descriptive than just `name`.  Often people prepend an identifier for their app, for example.